ngx_http_sxg_filter_module.c

```diff
@@ -408,6 +530,7 @@ static ngx_int_t ngx_http_sxg_header_filter(ngx_http_request_t* req) {
         invoke_subrequests(&value[i].value, req, ctx);
       }
     }
+    value->value = ctx->link_header;
```

Comment:
Should this be inside the `if` at line 530? Currently, it looks like it would overwrite every header value with the list non-preload links.